api/main.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from views import blueprint
+from database import db
+
+# initialize the flask app
+app = Flask(__name__)
+
+# load config values from the file at the path stored in an environment variable
+app.config.from_envvar('CRYFALL_CONFIG_FILE')
```

Comment:
This means "load the path to my config file from the environment variable `CRYFALL_CONFIG_FILE`"

Reply:
https://flask.palletsprojects.com/en/1.1.x/api/#flask.Config